### PR TITLE
Update exonum doc links to 0.10 version

### PR DESCRIPTION
**Pull request by MakarovS**
## Overview
Update exonum.com/doc links and use version `0.10` instead of `latest`.

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core/rust/ejb-app/TUTORIAL.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ of the Contribution Guide.
 You also need a ready-to-use Exonum Java service. You can use 
 [cryptocurrency-demo][cryptocurrency-demo] as an example, and find information about 
 implementing your own Exonum service 
-in the [user guide](https://exonum.com/doc/version/latest/get-started/java-binding/).
+in the [user guide](https://exonum.com/doc/version/0.10/get-started/java-binding/).
```
Comment (bullet-tooth) [inline exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: 404 for https://exonum.com/doc/version/0.10/get-started/java-binding/ ;(

Comment (MakarovS) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: Sure, 0.10 is not posted yet
Will add `[DO-NOT-MERGE]`

Comment (dmitry-timofeev) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: @MakarovS Have you confirmed the path the docs will be available at?

If/when yes, we will merge into `v0.4-release` branch, not waiting for the publication.

Comment (MakarovS) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: Yes, @Maria-Nosyk confirmed that it'd be `0.10`

File: exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 /**
  * Data of a transaction message in JSON. Matches the
```
Comment (dmitry-timofeev) [inline exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java]: I think "matches Exonum format" part is obsolete (they are now passed as hex).

Do we still need this class?

File: exonum-java-binding/core/rust/ejb-app/TUTORIAL.md (hunk 1 above)
Comment (dmitry-timofeev) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: Wonderful, please rebase then onto the release branch (we will sync it with master shortly after release).

File: exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java (hunk 2 above)
Comment (dmitry-timofeev) [reply exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java]: The downsides of suppressing warnings :-)

File: exonum-java-binding/core/rust/ejb-app/TUTORIAL.md (hunk 1 above)
Comment (MakarovS) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: We want Javadocs to include updated links, right?

Comment (dmitry-timofeev) [reply exonum-java-binding/core/rust/ejb-app/TUTORIAL.md]: Sure, that’s why we will publish the updated versions on our website once we fix all issues and make a release.

File: exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java (hunk 2 above)
Comment (dmitry-timofeev) [reply exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java]: :arrow_up_small: — it does not seem to be used.

**Pull request merged**